scripts/stablex/place_margin_orders.js

```diff
@@ -0,0 +1,91 @@
+const BatchExchange = artifacts.require("BatchExchange")
+const { sendTxAndGetReturnValue } = require("../../test/utilities.js")
+const BN = require("bn.js")
+
+const argv = require("yargs")
+  .option("tokens", {
+    describe: "Collection of trusted tokens",
+  })
+  .option("accountId", {
+    describe: "Account index of the order placer",
+  })
+  .option("margin", {
+    type: "float",
+    describe: "Percentage increase required for trade (fees not accounted)",
+    default: 0.2,
+  })
+  .option("sellAmount", {
+    type: "float",
+    describe: "Maximum sell amount (considered infinite if empty)",
+    default: 100,
```

Comment:
Why not infinite?

Reply:
Infinite will become a problem when trying to evaluate the margin (which will result in overflow). This number needs to be finite and we need some assertion regarding overflow of the `buyAmount = sellAmount * (1 + margin)`

I do, however, agree that 100 is low (I am currently using this as a default for my own testing purposes).